silx/gui/dialog/ColormapDialog.py

```diff
@@ -866,6 +866,12 @@ def __init__(self, parent=None, title="Colormap Dialog"):
         self._gammaSpinBox = qt.QDoubleSpinBox(parent=self)
         self._gammaSpinBox.setEnabled(False)
         self._gammaSpinBox.setRange(0., 1000.)
+        self._gammaSpinBox.setDecimals(4)
+        if hasattr(qt.QDoubleSpinBox, "setStepType"):
+            # Introduced in Qt 5.12
+            self._gammaSpinBox.setStepType(qt.QDoubleSpinBox.AdaptiveDecimalStepType)
+        else:
+            self._gammaSpinBox.setSingleStep(0.1)
         self._gammaSpinBox.setSingleStep(0.1)
```

Comment:
Looks to be a typo here: duplicated call to `setSingleStep`.

Reply:
oups

Reply:
Commit fixed.